README.md

```diff
@@ -101,6 +105,8 @@ const defaults = {
   // pngs are explicitly supported.
   imgTagExtensions: ['jpg', 'jpeg', 'png'],
 
+  imgTagExtensions: ["jpg", "jpeg", "png"],
```

Comment:
Duplicate imgTagExtensions property, you might still wanna get rid of that.

Reply:
Oops. Sure do!

Reply:
Ok. I just changed it to use `"` instead of `'` to match all the others.